.circleci/config.yml

```
@@ -75,16 +92,28 @@ jobs:
       - run:
           <<: *debian
 
+      - restore_cache:
+          keys:
+              - doc-{{ .Branch }}
+
       - run:
           name: build docs
           no_output_timeout: 25m
           command: |
             . venv/bin/activate
             export SHELL=$(which bash)
-            python -u runtests.py -g -j2 --doc html-scipyorg --doc latex
+            python setup.py install
+            cd doc
+            make html-scipyorg
+            make latex
             make -C doc/build/latex all-pdf LATEXOPTS="-file-line-error -halt-on-error"
             cp -f doc/build/latex/scipy-ref.pdf doc/build/html-scipyorg/
```

With your changes you are in the `doc` dir so you need to do either (my preference):
```suggestion
            make -C doc html-scipyorg
            make -C doc latex
            make -C doc/build/latex all-pdf LATEXOPTS="-file-line-error -halt-on-error"
            cp -f doc/build/latex/scipy-ref.pdf doc/build/html-scipyorg/
```
or
```
            cd doc
            make html-scipyorg
            make latex
            make -C build/latex all-pdf LATEXOPTS="-file-line-error -halt-on-error"
            cp -f build/latex/scipy-ref.pdf doc/build/html-scipyorg/
```

---

Thanks for your help! Let's give this a try then 😄